eras/alonzo/impl/cardano-ledger-alonzo.cabal

```
@@ -102,6 +102,7 @@ library testlib
         Test.Cardano.Ledger.Alonzo.Binary.RoundTrip
         Test.Cardano.Ledger.Alonzo.CostModel
         Test.Cardano.Ledger.Alonzo.ImpTest
+        Test.Cardano.Ledger.Alonzo.Binary.CostModelsSpec
```

Nothing important, just alphabetic grouping.
```suggestion
        Test.Cardano.Ledger.Alonzo.Binary.CostModelsSpec
        Test.Cardano.Ledger.Alonzo.Binary.RoundTrip
        Test.Cardano.Ledger.Alonzo.CostModel
        Test.Cardano.Ledger.Alonzo.ImpTest
```

---

It is important to prevent the descent into chaos :D Thank you for noticing